template/nuxt/nuxt.config.js

```diff
@@ -86,6 +86,7 @@ module.exports = {
   /*
   ** Nuxt.js dev-modules
   */
+
```

Comment:
Don't think this carriage return is needed as it's not done for the other comments on top of properties.
```suggestion
```